src/miniscript/context.rs

```diff
@@ -45,6 +45,9 @@ pub enum ScriptContextError {
     /// Only Compressed keys allowed under current descriptor
     /// Segwitv0 fragments do not allow uncompressed pubkeys
     CompressedOnly(String),
+    /// XOnly keys are only allowed in Tap context
+    /// The first element is key, and second element is current script context
+    XOnlyKeysNotAllowed(String, String),
```

Comment:
In cbd02d4910e56c01995152c27624a5f9d3da06e5:

I think the second `String` could be a `&'static str`